components/lcd/esp_lcd_gc9a01/include/esp_lcd_gc9a01.h

```diff
@@ -62,16 +62,17 @@ esp_err_t esp_lcd_new_panel_gc9a01(const esp_lcd_panel_io_handle_t io, const esp
  *
  * @param[in] sclk SPI clock pin number
  * @param[in] mosi SPI MOSI pin number
+ * @param[in] max_trans_sz Maximum transfer size in bytes
  *
  */
-#define GC9A01_PANEL_BUS_SPI_CONFIG(sclk, mosi)         \
-    {                                                   \
-        .sclk_io_num = sclk,                            \
-        .mosi_io_num = mosi,                            \
-        .miso_io_num = -1,                              \
-        .quadhd_io_num = -1,                            \
-        .quadwp_io_num = -1,                            \
-        .max_transfer_sz = SPI_LL_DMA_MAX_BIT_LEN >> 3, \
+#define GC9A01_PANEL_BUS_SPI_CONFIG(sclk, mosi, max_trans_sz)   \
```

Comment:
@Lzw655 @espzav Adding another parameter looks like a breaking API change. Should the version be changed to 2.0.0 instead of 1.2.0?

Reply:
Oh, I thought it didn't need to be changed since the 1.1.0 version has been marked yanked.

Reply:
@igrr Can we remove the 1.1.0 version from the ESP Registry?

Reply:
@tore-espressif Could you please help with this?

Reply:
>  I thought it didn't need to be changed since the 1.1.0 version has been marked yanked.

Breaking changes are allowed only between major releases, eg. 1.1.0 -> 2.0.0.

> Can we remove the 1.1.0 version from the ESP Registry?

1.1.0 was yanked. We want to avoid deleting broken versions, to keep sync between git and component-registry.

@Lzw655 Ideally, you can push a fix that would garantuee compatibility -> 1.2.1. Or we can re-release this as 2.0.0

Reply:
> > I thought it didn't need to be changed since the 1.1.0 version has been marked yanked.
> 
> Breaking changes are allowed only between major releases, eg. 1.1.0 -> 2.0.0.
> 
> > Can we remove the 1.1.0 version from the ESP Registry?
> 
> 1.1.0 was yanked. We want to avoid deleting broken versions, to keep sync between git and component-registry.
> 
> @Lzw655 Ideally, you can push a fix that would garantuee compatibility -> 1.2.1. Or we can re-release this as 2.0.0

Ok, I will submit a PR to bump their versions to 2.0.0.